test/test_cli.py

```diff
@@ -66,5 +67,5 @@ def test_hello_world(self, tmpdir, runner, package_location, interpreter):
             assert output_file.exists()
 
             # now run the produced zipapp
-            with subprocess.Popen([output_file], stdout=subprocess.PIPE) as proc:
-                assert proc.stdout.read().decode() == 'hello world\n'
+            with subprocess.Popen([str(output_file)], stdout=subprocess.PIPE, shell=True) as proc:
```

Comment:
I've opted for Path(...).as_posix() elsewhere, should we convert all instances to `str(Path(...))` for windows compatibility? Or does it not matter? (I'm not a windows expert by any stretch of the imagination)